src/encoded/static/components/item-pages/QualityMetricView.js

```diff
@@ -74,32 +44,49 @@ class QualityMetricViewOverview extends React.PureComponent {
         };
     }
 
+    static getSchemaQCFieldsAsOrderedPairs = memoize(function (schemaProp, context, schemas) {
```

Comment:
I would generally memoize an instance of this function rather than the static method itself, in case more than one instance of the class exists in a view, but I think is fine here since is not the case (i.e. only 1 view is active at a time).

Reply:
Updated.